gef.py

```diff
@@ -9962,12 +9963,15 @@ def tmux_setup(self) -> None:
         forcing the context to be redirected there."""
         tmux = which("tmux")
         ok("tmux session found, splitting window...")
-        old_ptses = set(os.listdir("/dev/pts"))
-        gdb.execute(f"! {tmux} split-window -h 'clear ; cat'")
+        
+        pane_info = subprocess.check_output([tmux, "splitw", "-h", '-F#{session_name}:#{window_index}.#{pane_index}-#{pane_tty}', "-P"]).decode().strip().split("-")
+        pane = pane_info[0]
+        pty = pane_info[1]
+        atexit.register(lambda : subprocess.run([tmux, "kill-pane", "-t", pane]))
+        
+        gdb.execute(f"! {tmux} send-keys -t {pane} 'clear;sleep infinity' C-m")
```

Comment:
Can you add a comment explaining why you need these keys to be sent?

Reply:
> Can you add a comment explaining why you need these keys to be sent?

okay, I add a comment and still use `cat`.